social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOC_People_Activity_CommentTestIT.java

```diff
@@ -200,6 +200,7 @@ public void test02_CommentOnYourFriendsActivity() {
     $(byId("cke_CommentTextarea" + id)).waitUntil(Condition.appears, Configuration.timeout).click();
     executeJavaScript("CKEDITOR.instances.CommentTextarea" + id + ".insertText(\"" + comment + "\")", "");
     // click on the button comment
+    Utils.pause(2000);
```

Comment:
yes it should work with " $(byText("Comment")).waitUntil(Condition.enabled, Configuration.timeout).click();" but sometimes the click is done before the button is enabled. It works without Utils.pause(2000) but the percentage of success of the action is not 100%